.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,28 +4,48 @@ about: Create a report to help us improve
 
 ---
 
+**IN ORDER TO ASSIST YOU, PLEASE PROVIDE THE FOLLOWING:**
+
+- Speech SDK log taken from a run that exhibits the reported issue. 
+See [instructions on how to take logs](https://docs.microsoft.com/azure/cognitive-services/speech-service/how-to-use-logging).
+
+- A stripped down, simplified version of your source code that exhibits the issue.
+
+- If relevant, a WAV file of your input audio.
+
+- Additional information as shown below
+
 **Describe the bug**
-A clear and concise description of what the bug is.
+
+A clear and concise description of what the bug is. If things are not working as you expect, 
```

Comment:
can you think of a way to word this...

Can you reproduce the problem with our samples, or provide a minimally modified version of our samples that would reproduce the problem?